client/index.html

```diff
@@ -96,7 +96,7 @@ <h1 class="title">Sign in to The Lounge</h1>
 							</div>
 							<div class="col-xs-12">
 								<label class="remember">
-									<input type="checkbox" name="remember" id="sign-in-remember" checked>
+									<input type="checkbox" name="remember" value="on" id="sign-in-remember" checked>
```

Comment:
Just explicitly stating the value to send (for reader's sanity), and I compare against it on the server.